client/keys/add_ledger_test.go

```diff
@@ -107,9 +112,10 @@ func Test_runAddCmdLedger(t *testing.T) {
 
 	require.NoError(t, cmd.ExecuteContext(ctx))
 
-	// Now check that it has been stored properly
-	kb, err := keyring.New(sdk.KeyringServiceName(), keyring.BackendTest, kbHome, mockIn)
+	kb, err := keyring.New(sdk.KeyringServiceName(), keyring.BackendTest, kbHome, mockIn, encCfg.Codec)
```

Comment:
```suggestion
	// Now check that it has been stored properly
	kb, err := keyring.New(sdk.KeyringServiceName(), keyring.BackendTest, kbHome, mockIn, encCfg.Codec)
```
let's keep the old comment